src/App.tsx

```diff
@@ -1,27 +1,41 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { GoodsList } from './GoodsList';
+import { get5First, getAll, getRedGoods } from './api/goods';
+import { Good } from './types/Good';
 
-// import { getAll, get5First, getRed } from './api/goods';
-// or
-// import * as goodsAPI from './api/goods';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <h1>Dynamic list of Goods</h1>
 
-    <button type="button" data-cy="all-button">
-      Load all goods
-    </button>
+export const App: React.FC = () => {
+  const [goods, setGoods] = useState<Good[]>([]);
 
-    <button type="button" data-cy="first-five-button">
-      Load 5 first goods
-    </button>
+  return (
+    <div className="App">
+      <h1>Dynamic list of Goods</h1>
 
-    <button type="button" data-cy="red-button">
-      Load red goods
-    </button>
+      <button
+        type="button"
+        data-cy="all-button"
+        onClick={() => getAll().then(allGoods => setGoods(allGoods))}
+      >
+        Load all goods
+      </button>
 
-    <GoodsList goods={[]} />
-  </div>
-);
+      <button type="button"
+              data-cy= "first-five-button"
+              onClick={() => get5First().then(first5Good => setGoods(first5Good))}>Load 5 first goods
+      </button>
```

Comment:
The same here as above. Aslo don't forget about code style
```suggestion
      <button
        type="button"
        data-cy= "first-five-button"
        onClick={() => get5First().then(first5Good => setGoods(first5Good))}
      >
        Load 5 first goods
      </button>
```